mobify/vellum
# Reorganize things for Adaptive 2.0

Separates out variables from base styles. Renames files called “vellum” to “base” to differentiate them from variables and clarify their purpose.

Status: **Ready to merge**

Reviewers: @jeffkamo @avelinet @cole-sanderson @mlworks @nastiatikk @yourpalsonja
Ticket: none
Linked PRs: [generator-adaptivejs#191](https://github.com/mobify/generator-adaptivejs/pull/191#discussion-diff-32174494)

## Changes
- Move variables to be a peer of _vellum.scss, which no longer imports it. Variables and base styles should/must now be imported separately. This allows variables to be the first import in project, ensuring that all subsequent files have access to project variables.
- Files/folders named “vellum” are now “base” to distinguish them from variables, and to help newcomers understand what they are. The library itself is still “Vellum”.
- Renamed `vellum/_global.scss` to `base/_general.scss` to avoid confusion between Adaptive 2.0’s global directory and this stylesheet.
- Some matching updates to test and support files.

## How to test-drive this PR
- Check out this branch
- Run `grunt` (default task)
- Open `test/index.html` in the browser and ensure nothing looks broken. Since no style code itself has changed, any breakages should be obvious (missing styles for example).

## Feedback
- [x] +1 @avelinet
- [x] +1 @jeffkamo

## Research resources
- none
